Vereinsstatuten.md

```diff
@@ -1,9 +1,9 @@
 
-# Statuten des Vereins DotNetDevs.at
+# Statuten des Vereins DotNetDevs.at - Verein zur Unterstützung von .NET Entwickler*innen in Österreich
 
 ## § 1: Name, Sitz und Tätigkeitsbereich
 
-(1) Der Verein führt den Namen "DotNetDevs.at".
+(1) Der Verein führt den Namen "DotNetDevs.at - Verein zur Unterstützung von .NET Entwickler*innen in Österreich".
```

Comment:
großes I nicht *, https://bildung.bmbwf.gv.at/ministerium/rs/formulieren_folder2012_7108.pdf?4e4zxz

Reply:
@dasMulli 	Ok, langsam kenn ich mich mit der Schreibweise nicht mehr aus.

Reply:
Bei Sternchen gehört dann ein kleines `i` - https://de.wikipedia.org/wiki/Gendersternchen

Reply:
Ihr gebt mir unterschiedliche Anforderungen. ;-)
Welche Schreibweise nehmen wir jetzt?

Reply:
Im Wikipedia Artikel von steht auch, dass der Genderstern von den meisten Vorleseprogrammen, z.B. für Sehbehinderte, nicht korrekt berücksichtigt wird und gilt daher als nicht barrierefrei. Ich bin deshalb für den Vorschlag mit dem Binnen-I, ohne dem Genderstern.

Reply:
Kategorie von "dingen die nicht in einen tweet passen" ^^
Mir ist es nicht soo wichtig, ich fänd es nur nett; schließe mich euerer Entscheidung dann an. Bin eher fan von komplett neutaler Kommunikation wenn notwendig (dadurch entfallen auch Barriere-hürden) und dann kann der screen-reader auch 1x einen stern vorlesen.